.github/workflows/backend_integration_tests_pr.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   test:
-    if: ${{ github.event.review.state == 'APPROVED' || (github.event_name == 'pull_request' && github.base_ref == 'staging') }}
+    if: ${{ github.event.review.state == 'APPROVED' || (github.event_name == 'pull_request' && github.base_ref == 'staging') || github.event_name == 'workflow_dispatch' }}
```

Comment:
extra: in order to check if the tests are passing I've added this line to be able to call them manually

Reply:
Test passed at https://github.com/yeagerai/genlayer-simulator/actions/runs/10093412255